projects/packages/backup/src/js/components/backup-video-section/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { ActionButton, getRedirectUrl } from '@automattic/jetpack-components';
+import { useProductCheckoutWorkflow, useConnection } from '@automattic/jetpack-connection';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import backupVideoThumbnail from './images/jetpack-backup-video-thumbnail.png';
+
+import './style.scss';
+
+const BackupVideoSection = ( {
+	siteProductAvailabilityHandler,
+	apiRoot,
+	apiNonce,
+	registrationNonce,
+} ) => {
+	const { run: handleCheckoutWorkflow, hasCheckoutStarted } = useProductCheckoutWorkflow( {
+		productSlug: 'jetpack_backup_t1_yearly',
+		redirectUrl: 'admin.php?page=jetpack-backup',
+		siteProductAvailabilityHandler,
+		from: 'jetpack-backup',
+	} );
+
+	const { siteIsRegistering, userIsConnecting, isOfflineMode, registrationError } = useConnection( {
+		registrationNonce: registrationNonce,
+		redirectUri: 'admin.php?page=jetpack-backup',
+		apiRoot: apiRoot,
+		apiNonce: apiNonce,
+		autoTrigger: false,
+		from: 'jetpack-backup',
+	} );
+
+	const errorMessage = isOfflineMode
+		? createInterpolateElement( __( 'Unavailable in <a>Offline Mode</a>', 'jetpack-backup-pkg' ), {
+				a: (
+					<a
+						href={ getRedirectUrl( 'jetpack-support-development-mode' ) }
+						target="_blank"
+						rel="noopener noreferrer"
+					/>
+				),
+		  } )
+		: undefined;
```

Comment:
This plus the action button was modeled after [this](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/connection/components/connect-screen/required-plan/index.jsx#L36) and [this](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/connection/components/connect-screen/required-plan/visual.jsx#L59)